### PR TITLE
docker: Add support for Google Translate v3

### PR DESCRIPTION
**Pull request by josh-english-2k18**
New logic to obtain Google Translation API v3 environment variables from the docker compose environment (see https://github.com/WeblateOrg/docker-compose/pull/109).

## Proposed changes

Set variables defined by proposed change to the docker compose environment.

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

## Other information

Trivial change.

### CODE REVIEW

**Comment by nijel** [comment]
Can you please add that to the documentation as well? (docs/admin/install/docker.rst)

**Comment by josh-english-2k18** [comment]
Updated documentation as per request.

**Comment by josh-english-2k18** [comment]
Please note that this PR (https://github.com/WeblateOrg/docker-compose/pull/109) goes along with these changes.

File: docs/admin/install/docker.rst [hunk 1]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
 
     Enables :ref:`google-translate` and sets :setting:`MT_GOOGLE_KEY`
 
+.. envvar:: WEBLATE_MT_GOOGLE_CREDENTIALS
+
+    Enables :ref:`google-translate-api-v3-advanced` and sets :setting:`MT_GOOGLE_CREDENTIALS`
```
Comment (nijel) [inline docs/admin/install/docker.rst:788]:
```suggestion
    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_CREDENTIALS`
```

File: docs/admin/install/docker.rst [hunk 2]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
...
+
+.. envvar:: WEBLATE_MT_GOOGLE_PROJECT
+
+    Enables :ref:`google-translate-api-v3-advanced` and sets :setting:`MT_GOOGLE_PROJECT`
```
Comment (nijel) [inline docs/admin/install/docker.rst:792]:
```suggestion
    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_PROJECT`
```

File: docs/admin/install/docker.rst [hunk 3]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
...
+
+.. envvar:: WEBLATE_MT_GOOGLE_LOCATION
+
+    Enables :ref:`google-translate-api-v3-advanced` and sets :setting:`MT_GOOGLE_LOCATION`
```
Comment (nijel) [inline docs/admin/install/docker.rst:796]:
```suggestion
    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_LOCATION`
```

File: docs/admin/install/docker.rst [hunk 4]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
 
     Enables :ref:`google-translate` and sets :setting:`MT_GOOGLE_KEY`
 
+.. envvar:: WEBLATE_MT_GOOGLE_CREDENTIALS
+
+    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_CREDENTIALS`
```
Comment (nijel) [inline docs/admin/install/docker.rst:788]:
```suggestion
    Enables :ref:`google-translate-api3` and sets :setting:`MT_GOOGLE_CREDENTIALS`
```

File: docs/admin/install/docker.rst [hunk 5]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
...
+
+.. envvar:: WEBLATE_MT_GOOGLE_PROJECT
+
+    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_PROJECT`
```
Comment (nijel) [inline docs/admin/install/docker.rst:792]:
```suggestion
    Enables :ref:`google-translate-api3` and sets :setting:`MT_GOOGLE_PROJECT`
```

File: docs/admin/install/docker.rst [hunk 6]
```diff
@@ -783,6 +783,18 @@ Machine translation settings
...
+
+.. envvar:: WEBLATE_MT_GOOGLE_LOCATION
+
+    Enables :ref:`google-translate-api-v3` and sets :setting:`MT_GOOGLE_LOCATION`
```
Comment (nijel) [inline docs/admin/install/docker.rst:796]:
```suggestion
    Enables :ref:`google-translate-api3` and sets :setting:`MT_GOOGLE_LOCATION`
```

**Pull request merged**